.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -1,22 +1,17 @@
 ---
-name: Bug Report (.NET Framework)
+name: Bug Report
 about: Create a report to help us improve
 labels: bug
 ---
------------------Bug Template--------------------------------------------------------------
 
-* SDK version: 
-* .NET runtime version (.NET TargetFramework in the .csproj file):
-* Hosting Info (IIS/Azure WebApps/etc): 
-* Platform and OS version: 
+* List of [NuGet packages](https://github.com/microsoft/ApplicationInsights-dotnet#nuget-packages) and version that you are using:
+* Runtime version (e.g. net461, net48, netcoreapp2.1, netcoreapp3.1, you can find this information from the `*.csproj` file):
+* Application type (Console/ASP.NET/ASP.NET Core):
+* Hosting environment (e.g. Windows Server 2016 + IIS, Ubuntu 20.04 LTS self-hosted, Azure WebApp):
 
 **Steps to reproduce.**
 Describe exactly how to reproduce the error. Include a code sample if applicable.
```

Comment:
Could we suggest that people create a github repository with a sample project reproducing the error?
If there's a polite way to ask this, it would dramatically improve our ability to investigate reported issues.

Reply:
That's too much ask IMHO.